pointm_test.go

```diff
@@ -0,0 +1,95 @@
+package geom_test
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"testing"
+
+	"github.com/go-spatial/geom"
+)
+
+func TestPointMSetter(t *testing.T) {
+	type tcase struct {
+		point    [3]float64
+		setter   geom.PointMSetter
+		expected geom.PointMSetter
+		err      error
+	}
+
+	fn := func(tc tcase) func(*testing.T) {
+		return func(t *testing.T) {
+			err := tc.setter.SetXYM(tc.point)
+			if tc.err == nil && err != nil {
+				t.Errorf("error, expected nil got %v", err)
+				return
+			}
+			if tc.err != nil {
+				if tc.err.Error() != err.Error() {
+					t.Errorf("error, expected %v got %v", tc.err, err)
+				}
+				return
+			}
+
+			// compare the results
+			if !reflect.DeepEqual(tc.expected, tc.setter) {
+				t.Errorf("setter, expected %v got %v", tc.expected, tc.setter)
+				return
+			}
+			xym := tc.setter.XYM()
+			if !reflect.DeepEqual(tc.point, xym) {
+				t.Errorf("XYM, expected %v, got %v", tc.point, xym)
+			}
+		}
+	}
+	testcases := []tcase{
```

Comment:
Let's keep this consistent, and call it `tests`.

Reply:
@gdey Done. BTW, I was based on `point` type tests where the variable was called `testcases`. I thought to change to `tests` the naming even there, let me know if it's fine or not.

Reply:
Yes, please. I'm not always good about being consistent. So, any help making sure things are consistent is always appriciated.